.github/workflows/python_ci.yml

```diff
@@ -0,0 +1,31 @@
+name: python_ci
+
+on: [push, pull_request]
```

Comment:
Can you delete "pull_request" here? We're only running workflows on push for now, will consider pull_request later due to security concerns.

Reply:
I can. But admins, as you just saw, have to approve first time contributors CI to run to negate this worry.

Reply:
Thanks. Yeah, I understand the risk is low especially with first-time confirmation. But it's also easy to revert broken changes if needed (there's a button) and we may eventually move this workflow onto the self-hosted runner which we're more concerned about.